Python script starting...
Traceback (most recent call last):
  File "D:\GitHub\stzlib\libraries\stzlib\test\temp\temp_8vrqsaml.py", line 102, in <module>
    count = traverse_and_replace(deep_list, "\u2665", "\u2605")
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\GitHub\stzlib\libraries\stzlib\test\temp\temp_8vrqsaml.py", line 76, in traverse_and_replace
    count += traverse_and_replace(lst[i], target, replacement)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\GitHub\stzlib\libraries\stzlib\test\temp\temp_8vrqsaml.py", line 76, in traverse_and_replace
    count += traverse_and_replace(lst[i], target, replacement)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\GitHub\stzlib\libraries\stzlib\test\temp\temp_8vrqsaml.py", line 76, in traverse_and_replace
    count += traverse_and_replace(lst[i], target, replacement)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  [Previous line repeated 996 more times]
RecursionError: maximum recursion depth exceeded
